### PR TITLE
Fixes Issue #20, Added 2 topics and an image for SDD.

### PR DESCRIPTION
**Pull request by Preeticp**
Added about_scenario_driven_planning.adoc,  using_scenario_driven_planning.adoc and sdd_wit_hierarchy.png for this PR. Also made minor additions to the Glossary.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-21-1 is available for testing at http://fabric8-online-docs-app-pr-21-fabric8-online-docs.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-21-2 is available for testing at http://fabric8-online-docs-app-pr-21-fabric8-online-docs.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-21-3 is available for testing at http://fabric8-online-docs-app-pr-21-fabric8-online-docs.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-21-4 is available for testing at http://fabric8-online-docs-app-pr-21-fabric8-online-docs.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-21-5 is available for testing at http://fabric8-online-docs-app-pr-21-fabric8-online-docs.badger.fabric8.io

File: user_guide/content/modules/about_scenario_driven_planning.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+* The associated Experience: The stuck developer identifies a team member who is online using the team pane in the IDE and sends a request for real-time collaboration. The other developer gets alerted and if accepted, they are presented with a code-editor that enables the two of them to edit the same code, in real time. An integration with the team’s preferred universal chat client enables discussion.
+. The Product Owner or Team Lead in sync with the Management identifies the Requirements or Features to actualize these Experiences and Value-Propositions. Bugs are tracked at the Requirements level. For example:
+* One of the Features for the previous scenario could be, a developer should be able to see a list of his teammates with their Online/Offline status within the IDE so that he may interact with them in real-time without leaving the IDE.
+. The Execution team along with the Product owner selects some of the work items for each Iteration and further breaks them down into Tasks which can be assigned to the team members.
+
+Fundamental and Paper-Cut work item types also follow a similar development workflow.
```
[the remark this answers is not in the document]
Comment (Preeticp) [reply user_guide/content/modules/about_scenario_driven_planning.adoc]: Agree, going by UX, it is Papercuts, will make the changes throughout.

File: user_guide/content/modules/about_scenario_driven_planning.adoc [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+A scenario determines tangible customer value and resolves issues they face keeping in the mind the broader product vision. It envisages the Value Propositions and Experiences required to deliver customer value. It proposes complete customer experiences to motivate customers to adopt the product.
+From the customer point of view, a scenario tries to answer the question “In your business, do you need to…?”.
+
+==== Fundamentals
+Fundamentals are Work items which ensure that quality metrics such as compatibility, security, compliance, performance, and so on are suitably met. These have associated Value Propositions and Experiences and follow the same development process as Scenarios.
+
+==== Papercuts
+Papercuts are a collection of low-priority bugs and small convenience features, which collectively, disrupt the user experience and have the potential to make the user lose interest in the product. These aggregated irritants follow the same development process as Scenarios.
+
+===== Value Proposition
```
[the remark this answers is not in the document]
Comment (Preeticp) [reply user_guide/content/modules/about_scenario_driven_planning.adoc]: Value Proposition and Experiences are second level of WIs within the Portfolio bucket. Scenarios, Fundamentals and Papercuts are broken down into Value Proposition and Experiences.  So I was trying to depict the hierarchy by making it 5th level. Does that work?

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-21-6 is available for testing at http://fabric8-online-docs-app-pr-21-fabric8-online-docs.badger.fabric8.io

File: user_guide/content/modules/about_scenario_driven_planning.adoc [hunk 3]
```diff
@@ -0,0 +1,58 @@
+[#about_scenario_driven_planning]
+= About Scenario Driven Planning
+
+One of the development methodologies supported by the Planner is the Scenario driven development approach to plan and track your development work. The Scenario driven development process uses the following work item type hierarchy:
```
Comment (mishaone) [inline user_guide/content/modules/about_scenario_driven_planning.adoc]: A lot of published work about various methodologies uses this format for the name: Xyz-driven development (e.g. Test-driven development, Behavior-driven development) so I'd recomment s/Scenario driven development/Scenario-driven development everywhere.

Comment (rkratky) [reply user_guide/content/modules/about_scenario_driven_planning.adoc]: +1

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-21-7 is available for testing at http://fabric8-online-docs-app-pr-21-fabric8-online-docs.badger.fabric8.io

**Comment by Preeticp** [comment]
Resolved all review comments.

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-21-8 is available for testing at http://fabric8-online-docs-app-pr-21-fabric8-online-docs.badger.fabric8.io

**Comment by Preeticp** [comment]
@mishaone could you please submit your review comments again? I was able to see only one comment, which I have addressed.

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-21-9 is available for testing at http://fabric8-online-docs-app-pr-21-fabric8-online-docs.badger.fabric8.io

**Comment by fabric8cd** [comment]
@Preeticp snapshot fabric8/fabric8-online-docs:SNAPSHOT-PR-21-10 is available for testing at http://fabric8-online-docs-app-pr-21-fabric8-online-docs.badger.fabric8.io

**Comment by Preeticp** [comment]
@rkratky, @mishaone I have addressed all the review comments. Please let me know if this can be merged.
Robert about the Value Proposition and Experiences in Portfolio, I had kept it at H5 to depict that they are the next hierarchical level in Portfolio. However I saw that when rendered they seemed rather inconspicuous, so changed them back to H4 as suggested by you.

**Pull request merged**